### DISCUSSION TITLE
1.42 - mo' problems, mo' fixes

### DISCUSSION THREAD

**Discussion by cerebrate** [arkane-systems/genie#165] [General]
* Regression fixes.

<hr /><em>This discussion was created from the release <a href='https://github.com/arkane-systems/genie/releases/tag/v1.42'>1.42 - mo' problems, mo' fixes</a>.</em>

**Comment by tomdecouttere** [comment]
genie --version 
doesn't work anymmore

tom@CMEC-LT-001-wsl:~$ genie -i
tom@CMEC-LT-001-wsl:~$ genie --version
--version option cannot be combined with other arguments.

**Reply by cerebrate** [reply]
Known issue, alas. Please see https://github.com/arkane-systems/genie/issues/173.
